server/src/main/java/org/tctalent/server/util/locale/LocaleHelper.java

```diff
@@ -112,6 +119,22 @@ public class LocaleHelper {
         return cts;
     }
 
+    /**
+     * Computes an OffsetDateTime from a LocalDate, LocalTime and timezone
+     * @param localDate Local date
+     * @param localTime Local time
+     * @param timezone Timezone string - if null or blank UTC timezone is used
+     * @return An OffsetDateTime
+     */
+    public static OffsetDateTime getOffsetDateTime(
+        LocalDate localDate, LocalTime localTime, @Nullable String timezone) {
+        ZoneOffset offset = ZoneOffset.UTC;
+        if (!ObjectUtils.isEmpty(timezone)) {
+            offset = ZoneId.of(timezone).getRules().getOffset(Instant.now());
+        }
```

Comment:
Handle a possible Runtime exception here. If so, does it make sense to fall back to UTC and continue processing:
```
    if (!ObjectUtils.isEmpty(timezone)) {
        try {
            offset = ZoneId.of(timezone).getRules().getOffset(Instant.now());
        } catch (DateTimeException e) {
            System.err.println("Invalid timezone provided: " + timezone + ". Falling back to UTC.");
        }
    }
```

Reply:
Nice one - added it in (with unit test!)